modules/rules_erlang/3.8.5/MODULE.bazel

```diff
@@ -0,0 +1,40 @@
+module(
+    name = "rules_erlang",
+    compatibility_level = 3,
+    version = "3.8.5",
+)
+
+bazel_dep(
+    name = "rules_pkg",
+    version = "0.5.1",
+)
```

Comment:
Just a drive-by comment: Could this be a dev dependency or do you use it outside the release process?

Reply:
It's used `source_archive.bzl` for the `source_archive` macro, so I don't believe it's a dev dependency. But thanks for pointing it out. If I have a misunderstanding of how `dev_dependency` is supposed to be used, I'd be happy to learn more.